mitodl/bootcamp-ecommerce
# chore: upgrade python to 3.10.13

#### What are the relevant tickets?
https://github.com/mitodl/bootcamp-ecommerce/issues/1360

#### What's this PR do?
Upgrade python to 3.10.13

#### How should this be manually tested?
- Build and start the docker container.
- Go into the docker container and check the python version which should be 3.10.13
- Run the tests to see if the project is functional using `pytest --no-pylint` command inside the container

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
